src/transformer.ts

```diff
@@ -122,6 +122,10 @@ export default class Transformer {
         this.hasJson = true;
 
         result.push(this.wrapWithZodValidators('jsonSchema', field, inputType));
+      } else if (inputType.type === 'True') {
+        result.push(
+          this.wrapWithZodValidators('z.literal(true)', field, inputType),
```

Comment:
What's the difference between a boolean and a literal when using Zod?

Reply:
It's valid only true value. You can not use false value in this field.

Reply:
Thank you for the explanation. You are exactly right.